### PR TITLE
Updated Episode 12 to fix misleading title and add a little more information

### PR DESCRIPTION
**Pull request by emcaulay**
Hi,

I am submitting a pull request with a few changes to Episode 12.

1. Change title. (I think there was an issue about this topic and I will find it and link it here). The title was "Transformations - Exporting," which gives the impression that this is a continuation of the Transformations topic from Episodes 10 and 11. However, that's not accurate. The only thing discussed in the episode is how to export data from OpenRefine, and you don't need to do transformations to want to know how to export data. It's a distinct topic and activity. I think changing the title will make it easier for Instructors and Learners alike to understand the topic of the lesson.

2. I added a new paragraph at the beginning to highlight that OpenRefine doesn't "save" your work into the file that you got the data from. This characteristic is unlike a lot of software applications that are used to look at and edit data or text, so it's worth highlighting that this program works differently.

3. I added a new paragraph that discusses exporting a portion of your data by faceting and exporting when a facet is selected. 

I think this episode could be further developed, but that's all I have time for right now. And I think baby steps are okay!

I am teaching this lesson next week, and that's why I'm going over it so carefully. Thank you, maintainers, for your patience and help!

### CODE REVIEW

**Comment by emcaulay** [comment]
The issue #143 requests and suggests several changes. I have made progress in addressing some of those changes, but not all of the topics raised in that issue. 

I would recommend that we start a new issue that contains the topics that remain to be addressed since it can be really hard to parse through an issue with a lot of different suggestions in it.

File: _episodes/12-export-transformation.md [hunk 1]
```diff
@@ -1,16 +1,24 @@
 ---
-title: "Transformations - Exporting"
+title: "Exporting your revised data"
```
Comment (ostephens) [inline _episodes/12-export-transformation.md:2]: I think just "Exporting data" would be a better title - the data doesn't necessarily have to have been revised to do the export

File: _episodes/12-export-transformation.md [hunk 2]
```diff
@@ -1,16 +1,24 @@
...
-- "How do I export transformed data?"
+- "How do I export data from OpenRefine?"
 objectives:
 - "Explain how to export data in different formats from OpenRefine"
 keypoints:
 - "You can export your data in a variety of formats"
 ---
 
+## Note about OpenRefine
+All the edits you make to your data using OpenRefine are being stored inside the OpenRefine program and are not being saved to your original file. That's why OpenRefine uses the terms "import" and "export" to talk about moving your data in and out of the OpenRefine interface. Therefore, in order to save your work into a file format you can view in other programs or share with others, you need to export your data.
```
Comment (ostephens) [inline _episodes/12-export-transformation.md:14]: ❤️

File: _episodes/12-export-transformation.md [hunk 3]
```diff
@@ -1,16 +1,24 @@
...
+
 ## Exporting data
 Once you have finished working with a data set in OpenRefine you may wish to export it. The export options are accessed through the ```Export``` button at the top right of the OpenRefine interface.
 
 Export formats support include HTML, Excel and comma- and tab-separated value (csv and tsv). You can also write a custom export, selecting to export specific fields, adding a header or footer and specifying the exact format.
+
+## Exporting a portion of your data
+You can also export a portion of your data by using facets or filters to select a portion of your data. With only those rows selected, you can select the export format and your resulting file will only include the select rows. 
+
+*Note well:* It's easy to export only a portion of your data by accident, so make sure you look at the top left to ensure all rows are being displayed when you want to do a full export.
```
Comment (ostephens) [inline _episodes/12-export-transformation.md:24]: If you use the custom tabular export you also get the option to override the current facets - but that may be too much detail here? Happy with this change as you've written it unless you feel this detail is worth adding

**Review by ostephens — COMMENTED** [review]
Thanks so much @emcaulay . My only substantive comment is about the episode title - if you are happy to amend that, we can get this merged

File: _episodes/12-export-transformation.md [hunk 4]
```diff
@@ -1,16 +1,24 @@
...
 - "You can export your data in a variety of formats"
 ---
 
+## Note about OpenRefine
+All the edits you make to your data using OpenRefine are being stored inside the OpenRefine program and are not being saved to your original file. That's why OpenRefine uses the terms "import" and "export" to talk about moving your data in and out of the OpenRefine interface. Therefore, in order to save your work into a file format you can view in other programs or share with others, you need to export your data.
+
 ## Exporting data
 Once you have finished working with a data set in OpenRefine you may wish to export it. The export options are accessed through the ```Export``` button at the top right of the OpenRefine interface.
 
 Export formats support include HTML, Excel and comma- and tab-separated value (csv and tsv). You can also write a custom export, selecting to export specific fields, adding a header or footer and specifying the exact format.
```
Comment (irennie) [inline _episodes/12-export-transformation.md:19]: Hope it's appropriate to suggest this here but would the phrase

> Export formats support include HTML, Excel and comma- and tab-separated value (csv and tsv).

work better as

> Export formats support include HTML, Excel and csv and tsv (comma- and tab-separated value).

This would make it clearer that the "-" here is a hyphen indicating an word fragment rather than a dash.

**Comment by ostephens** [comment]
@jas58 I think the work that @emcaulay has done here is really worth adding to the lesson although I still feel the episode title would be better as "Exporting data" rather than "Exporting your revised data" as per my comment above.

If you agree I'd suggest we can merge this and then edit the lesson title afterwards. Does that sound sensible to you?

**Comment by jas58** [comment]
Agree!  Revise title and merge.
